content/node/trustlessStaking.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Trustless Third Party Staking
+menuTitle: Trustless Third Party Staking
+weight: 30
+aliases:
+  - /home/node/trustlessStaking
+description: Trustless Third Party Staking utilizes non-custodial feature to stake pokt without handing over custody of tokens to the Node Operator at any stage.
+---
+
+Trustless Third Party Staking utilizes non-custodial feature to stake pokt without handing over custody of tokens to the Node Operator at any stage. With this method, Node Operators have no ability to steal either the stake or the rewards. Clients can also unstake without any intervention of the node operator and get their funds back.
+
+**Before proceeding, please consult with your preffered Node Operator if they support this method of staking.**
+
+This document will show how to stake a node with 15k tokens and later upstake it to 60k.
+We stake with 15k first to minimize any loss resulting from mistake(eg. Typing incorrect output address or public key) 
+
+Non-Custodial Staking method involves the following addresses/wallets
+
+**Output Address**
+Never share the private key of your output address. Make ample backups of this!
+You should already have this, a wallet with your funds. This can also be referred to as the non-custodial address.
+
+
+**Operator Address**
+An Operator Address will be created by the Node Operator.  Node operator will manage(edit Domain/Chains) and run your node using the key of the operator address. 
+
+The very first stake initiated from the output address can establish a link between the operator address and the output address. Node Operator can then start running your node using the operator address. Any node rewards will be sent to the output address. Additionally; if unstaked, the unstaked funds will also go to the output address. An output address can be linked to multiple operator addresses
+
+## Introduction
+
+**Staking Bins**:
+You can stake 15k,30k,45k,60k tokens. A 30k pocket node will earn the same as two 15k nodes for instance. It is a good practice to stake 10 tokens on top of that.
+See [stake-weighted-servicer-rewards](/learn/economics/nodes/#stake-weighted-servicer-rewards) for more info
+
+
+
+**How non-custodial looks like on Poktscan**
+
+https://poktscan.com/node/bab0bc034cbf4ae82a5069b677e8c2f39098e361
+![](/images/poktscan-noncustodial.png)
+
+Note the stake type “Non-Custodial”. Also note that the address of the node is different from the output address. In the example above:
+
+**Operator Address:**  bab0bc034cbf4ae82a5069b677e8c2f39098e361  
+**Operator Address Public Key:** d7a3f683341a5fd82adf4f4411544527ba15726d9558c5b83b9a2c6946b4df44  
+**Output Address:** 3574ca25de8744aa167cff234b69b8a4eef97711
+
+Notice there is a balance of 36.57 present. Node requires some pokt to pay to claim rewards and also any staking operations(edit domain/chain) node operator will do. Either you or the node operator can keep the operator account topped up periodically.
```

Comment:
> Node requires some pokt to pay to claim rewards

The node requires some POKT to pay for claiming rewards 

> any staking operations(edit domain/chain) node operator will do.

any staking operations the node operator will perform.

> Either you

Either the output address owner